README.md

```diff
@@ -40,10 +40,151 @@ Note: If you use the Drush command, it is advisable to Move (not copy) the insta
+communicate with Fedora. As such it does not work with Islandora's authentication system, meaning it can only be used to 
+display public datastreams, or it can be configured to give full access to show anything in the system. This does not 
+give any user the ability to access any datastream however, just the JPEG generated for display in the browser, not the 
+original.
+
+Additionally there are issues with it's caching system in that it needs to be manually flushed please read the project 
+[documentation](https://github.com/loris-imageserver/loris/blob/development/doc/cache_maintenance.md) for more 
+information.
+
+Provided below is instructions for installing loris on Ubuntu 14.04, the commands / packages may change slightly 
+depending on your setup.
+
+#### Create loris user
+```bash
+useradd -d /var/www/loris2 -s /sbin/false loris
+```
+
+#### Setup locale
+
+```bash
+locale-gen "en_US.UTF-8"
+update-locale LC_ALL="en_US.UTF-8" LANG="en_US.UTF-8"
+dpkg-reconfigure locales
+echo "export LC_ALL=\"en_US.UTF-8\"" >> /root/.bashrc
+echo "LC_ALL=\"en_US.UTF-8\"" >> /etc/environment
+echo "LANG=\"en_US.UTF-8\"" >> /etc/environment
+```
+
+#### Update packages and install tools
+```bash
+sudo apt-get update -y
+sudo apt-get install -y \
+            apache2 \
+            curl \
+            git \
+            libapache2-mod-wsgi \
+            libfreetype6 \
+            libfreetype6-dev \
+            libjpeg8 \
+            libjpeg8-dev \
+            liblcms2-2 \
+            liblcms2-dev \
+            liblcms2-utils \
+            libtiff5-dev \
+            python-dev \
+            python-pip \
+            python-setuptools \
+            unzip \
+            zlib1g-dev
+```
+
+#### Enable Apache Headers Module.
+```bash
+a2enmod headers expires
+```
+
+#### Install Kakadu
+```bash
+curl -o /usr/local/lib/libkdu_v74R.so -L https://github.com/loris-imageserver/loris/raw/development/lib/Linux/x86_64/libkdu_v74R.so
+chmod 755 /usr/local/lib/libkdu_v74R.so
+curl -o /usr/local/bin/kdu_expand -L https://github.com/loris-imageserver/loris/raw/development/bin/Linux/x86_64/kdu_expand
+chmod 755 /usr/local/bin/kdu_expand
+ln -s /usr/lib/`uname -i`-linux-gnu/libfreetype.so /usr/lib/
+ln -s /usr/lib/`uname -i`-linux-gnu/libjpeg.so /usr/lib/
+ln -s /usr/lib/`uname -i`-linux-gnu/libz.so /usr/lib/
+ln -s /usr/lib/`uname -i`-linux-gnu/liblcms.so /usr/lib/
+ln -s /usr/lib/`uname -i`-linux-gnu/libtiff.so /usr/lib/
+echo "/usr/local/lib" >> /etc/ld.so.conf && ldconfig
+```
+
+#### Install Python and Loris dependencies.
+```bash
+pip install --upgrade pip
+pip install Werkzeug configobj Pillow ordereddict requests mock responses
+```
+
+#### Clone Loris git repository (we currently recommend the development branch).
+```bash
+git clone https://github.com/loris-imageserver/loris.git /opt
+```
+
+#### Install Loris.
+```bash
+cd /opt/loris
+./setup.py install
```

Comment:
Instead of separately installing the dependencies and the server, might it make sense to condense it down into something like:
```bash
# May also be desirable to use the "-e" flag.
pip install git+https://github.com/loris-imageserver/loris.git#egg=loris
```
... `pip` will automatically grab the listed dependencies.